bitnami/discourse/templates/deployment-sidekiq.yaml

```
@@ -0,0 +1,72 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "discourse.fullname" . }}-sidekiq
+  labels:
+    {{- include "discourse.labels" . | nindent 4 }}
+    component: sidekiq
```

Instead of `component` use `app.kubernetes.io/component`

```suggestion
    app.kubernetes.io/component: sidekiq
```